planner/core/rule_join_reorder_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 PingCAP, Inc.
```

Comment:
Good job! But indeed this test is an integrated test. How about moving it to "integration test" and check the result of explain?

Reply:
done